tools/lint-commits

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This file is copied from the original tools/commit-message-lint at zulip/zulip,
+# Edited at Line 14 Col 97 (zulip -> python-zulip-api)
+# Please don't edit here; instead update the zulip/zulip copy and then resync this file.
+
+# Lint all commit messages that are newer than upstream/master if running
+# locally or the commits in the push or PR Gh-Actions.
+
+# The rules can be found in /.gitlint
+
+if [[ "
+$(git remote -v)
+" =~ '
+'([^[:space:]]*)[[:space:]]*(https://github\.com/|ssh://git@github\.com/|git@github\.com:)zulip/python-zulip-api(\.git|/)?\ \(fetch\)'
```

Comment:
Probably we should do a commit to zulip/zulip that makes the repository name a variable at the top of the file.

Reply:
Okay, we can do that.